GitCommands/Settings/AppSettings.cs

```diff
@@ -1259,6 +1259,12 @@ public static int RevisionGridQuickSearchTimeout
             set => SetInt("revisiongridquicksearchtimeout", value);
         }
 
+        public static bool ShowCommitBodyInRevisionGrid
+        {
+            get => GetBool("showcommitbodyinrevisiongrid", true);
+            set => SetBool("showcommitbodyinrevisiongrid", value);
```

Comment:
There is a mixed capitalization style in this file. For new settings, we use Delphi-Case:
```suggestion
            get => GetBool("ShowCommitBodyInRevisionGrid", true);
            set => SetBool("ShowCommitBodyInRevisionGrid", value);
```